style.css

```diff
@@ -677,3 +699,111 @@ form.send-message button:focus-visible {
   align-items: center;
   column-gap: 10px;
 }
+
+/* Modals Styles */
+.modal-section {
+  display: none;
+  position: fixed;
+  top: 0;
+  padding-top: 5%;
+  width: 100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.5);
+  z-index: 999;
+  justify-content: center;
+}
+
+.modal-section.active {
+  display: flex;
+}
+
+.modal-container {
+  width: 80%;
+  height: min-content;
+  background: #fff;
+  border-radius: 10px;
+  padding: 20px 24px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
```

Comment:
- [ ] For the desktop version of the modal, Kindly adjust the buttons to share the baseline with the image. It should be positioned at the bottom, but on the same baseline as the bottom of the image used. 

### Your design

![image](https://user-images.githubusercontent.com/77106962/180282602-3fc9a894-41ba-4792-afa0-5792dfe166c0.png)

### Design Template

![image](https://user-images.githubusercontent.com/77106962/180282709-8922e18f-7c7a-41f6-bb55-87a966fc283e.png)